docs/source/migration/4.0.mdx

```diff
@@ -9,27 +9,29 @@ description: From 3.x
 
 </Note>
 
-Apollo Kotlin 3.0 was a major rewrite of Apollo in Kotlin multiplatform.
+Apollo Kotlin 3 was a major rewrite of Apollo in Kotlin multiplatform.
 
-Apollo Kotlin 4.0 focuses on tooling, stability and fixing some API regrets that came with 3.x.
+Apollo Kotlin 4 focuses on tooling, stability and fixing some API regrets that came with 3.
 
-Because most of the common APIs stayed the same, we [kept the package name unchanged](https://github.com/apollographql/apollo-kotlin/issues/4710). Apollo Kotlin 4.0 removes some deprecated symbols. We strongly recommend removing deprecated usages before migrating to 4.0.  
+While most of the common APIs stayed the same, Apollo Kotlin 4 contains a few binary breaking changes. To account for that, and in order to be more future proof, we [changed the package name](https://github.com/apollographql/apollo-kotlin/issues/4710) to `com.apollographql.apollo`.
 
-If you are using a library that depends on Apollo Kotlin transitively, you need it to update to 4.x before you can update your own application to 4.0.
+Apollo Kotlin 4 removes some deprecated symbols. We strongly recommend removing deprecated usages before migrating to version 4.  
+
+If you are using a library that depends on Apollo Kotlin transitively, you need it to update to version 4 before you can update your own application to version 4.
```

Comment:
I forgot to move that part to next_docs. Note that that last line isn't true anymore under `com.apollographql.apollo`, I'll clean that up in a follow up PR